TravelExpenses/AppDelegate.swift

```diff
@@ -184,16 +184,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
-            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.reportStatus.localName, query: "/\(TravelexpenseMetadata.EntitySets.reportStatus.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
-            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.expenseItemAttachment.localName, query: "/\(TravelexpenseMetadata.EntitySets.expenseItemAttachment.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.payments.localName, query: "/\(TravelexpenseMetadata.EntitySets.payments.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.reservations.localName, query: "/\(TravelexpenseMetadata.EntitySets.reservations.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.expenses.localName, query: "/\(TravelexpenseMetadata.EntitySets.expenses.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.expenseItems.localName, query: "/\(TravelexpenseMetadata.EntitySets.expenseItems.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.tripItems.localName, query: "/\(TravelexpenseMetadata.EntitySets.tripItems.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.reservationItems.localName, query: "/\(TravelexpenseMetadata.EntitySets.reservationItems.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.expenseReports.localName, query: "/\(TravelexpenseMetadata.EntitySets.expenseReports.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.currencies.localName, query: "/\(TravelexpenseMetadata.EntitySets.currencies.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
+            try! offlineODataProvider.add(definingQuery: OfflineODataDefiningQuery(name: TravelexpenseMetadata.EntitySets.reportStati.localName, query: "/\(TravelexpenseMetadata.EntitySets.reportStati.localName)\(queryLimit)", automaticallyRetrievesStreams: false))
```

Comment:
The plural of "status" is "statuses", not "stati". [Merriam-Webster entry](https://www.merriam-webster.com/dictionary/status).